src/tools/perf/Makefile.am

```diff
@@ -23,9 +23,9 @@ ucx_perftest_SOURCES = \
 	perftest_run.c \
 	perftest_params.c
 
-ucx_perftest_CPPFLAGS = $(BASE_CPPFLAGS) $(RTE_CPPFLAGS)
+ucx_perftest_CPPFLAGS = $(BASE_CPPFLAGS)
```

Comment:
Also the same under `lib/Makefile.am`?

Reply:
Good catch. I updated and squashed to safe the time.